keyboards/kemove/boards/KEMOVE_SNOWFOX/v1_4/board.h

```diff
@@ -68,9 +68,9 @@
 #define LINE_LED2_CS                                    PAL_LINE(IOPORT1, 16)
 
 
-#define LINE_UART_RX                                    PAL_LINE(IOPORT1, 14)
-#define LINE_UART_TX                                    PAL_LINE(IOPORT1, 27)
-#define LINE_BLE_RSTN                                   PAL_LINE(IOPORT0, 20)
+#define LINE_UART_RX                                    PAL_LINE(IOPORT1, 26) // To pin 21 on BLE
+#define LINE_UART_TX                                    PAL_LINE(IOPORT1, 27) // To pin 20 on BLE
+#define LINE_BLE_RSTN                                   PAL_LINE(IOPORT0, 23) // Unsure, to pin 11 on BLE
```

Comment:
This should be correct. It's been a long time since I last worked on this, but I am 99% sure there's only three connection going between the MCU and the BLE module. This should be the reset pin.